### DISCUSSION TITLE
No File Upload submit location

### DISCUSSION THREAD

**Discussion by VanessaWoodson** [community/community#23178] [New to GitHub]
<p>There is no file upload area for me to publish. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/8/c88b8a5d3e95982ce89c9a751056ddafe8d0f6d2.jpeg" data-download-href="/uploads/short-url/sC6spk4hFiSnLqC2nJt5ind9MyK.jpeg?dl=1" title="IMG_0493" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/78329131/181075463-7ed38f1d-fef1-40f0-9a34-53e3ef256bec.jpeg" alt="IMG_0493" data-base62-sha1="sC6spk4hFiSnLqC2nJt5ind9MyK" width="375" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/8/c88b8a5d3e95982ce89c9a751056ddafe8d0f6d2_2_375x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/c/8/c88b8a5d3e95982ce89c9a751056ddafe8d0f6d2_2_562x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/8/c88b8a5d3e95982ce89c9a751056ddafe8d0f6d2_2_750x1000.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/8/c88b8a5d3e95982ce89c9a751056ddafe8d0f6d2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">IMG_0493</span><span class="informations">1512×2016 823 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>  Please help - Newbie

**Comment by billythekid** [comment]
<p>Hi <a class="mention" href="/vanessawoodson">@VanessaWoodson</a>, welcome to the forum!</p>
<p>Just next to the green button there’s an “Add file” button. If you click that you should get an “Upload files” option.</p>
<p><img src="https://user-images.githubusercontent.com/330170/181075466-e81d5f00-179e-4fea-8dd3-2b1a98505c13.png" alt="4e4cc513fcd07c136414bd139677fb3c6d22a07c" data-base62-sha1="ucIq9rkURZ8rbNvP7M1NsQGixk1" width="416" height="201"></p>
<p>Does that help?</p>
<p>Regards,<br>
Billy :cowboy_hat_face:</p>

**Reply by Lexgend13** [reply]
How do I even get to the screen Vanessa showed?

**Comment by VanessaWoodson** [comment]
<p>Yes thank you so much</p>
